documentation/queries/sql_queries.md

```diff
@@ -17,7 +17,7 @@ order by user_id
 ```sql
 select 
 	count(0)
-from .ubi_log_events
+from ubi_log_events
```

Comment:
@RasonJ I think you had the name of the store being `ubi_log`. I did not change these to `ubi_ubi_log_events` just for clarity. So just a note about that.